dev/com.ibm.ws.security.wim.adapter.ldap/src/com/ibm/ws/security/wim/adapter/ldap/LdapAdapter.java

```diff
@@ -284,7 +294,7 @@ public void initialize(Map<String, Object> configProps) throws WIMException {
         iLdapConfigMgr = new LdapConfigManager();
         iLdapConfigMgr.initialize(configProps);
 
-        iLdapConn = new LdapConnection(iLdapConfigMgr, kerberosService);
+        iLdapConn = new LdapConnection(iLdapConfigMgr, kerberosService, configAdminRef == null ? null : configAdminRef.getService());
```

Comment:
I don't think configAdminRef can ever be null.

Reply:
```suggestion
        iLdapConn = new LdapConnection(iLdapConfigMgr, kerberosService, configAdminRef.getService());
```

Reply:
Will change.